platforms-serverless/serverless-framework-lambda/tsconfig.json

```
@@ -5,6 +5,7 @@
     "outDir": ".",
     "strict": true,
     "lib": ["esnext", "dom"],
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "skipLibCheck": true,
```

To suppress a type issue in `lodash` itself.

---

Make this a comment then, it's ok to have a comment in tsconfig
```suggestion
	// workaround for a type issue in lodash itself
    "skipLibCheck": true,
```